src/FormGroup.tsx

```diff
@@ -3,12 +3,14 @@ import {get} from './constants'
 import sx, {SxProp} from './sx'
 import {ComponentProps} from './utils/types'
 
+/** @deprecated Use FormControl instead. See https://primer.style/react/FormControl for more details. */
```

Comment:
I don't think this will apply the annotation correctly as the assignment is part of internal scope. I imagine we'd need to deprecate the following line instead on line 26.

```js
export default Object.assign(FormGroup, {Label: FormGroupLabel})
```